concepts/default-enarx-trust-process.md

```diff
@@ -0,0 +1,480 @@
+# TBD: Default Enarx trust process
+- Authors: [Mike Bursell](mike@p2ptrust.org)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-03-17
+- Status Note: under discussion
+- Start Date: 2020-03-17 (date you started working on this idea)
+- Tags: trust
+- Addresses: (Issue #)[https://github.com/enarx/enarx/issues/]
+
+TODO: this is curently listed as a "concepts" document, but also
+includes design information and implementation requirements.  Need
+to decide where it best sits.  Neither "concepts" nor "features"
+(as described in the README.md) fit very well.
+
+## Summary
+
+Setting up trust domains and being able to transfer certain entities
+between them in central to Enarx.  This RFC describes the default
+trust process for Enarx and lays out sets of requirements for
+implementations.
+
+## Motivation
+
+One of the key aims of the Enarx project is to allow clients to run
+workloads on hosts which are "untrusted".  This RFC builds on previous
+RFCs to describe the default model by which Enarx allows this to take
+place, using the capabilities of TEEs, the underlying CPU(s) and
+firmware of the host and other parties.
+
+It should be noted that the key party for whom trust is considered
+important within the Enarx project is the workload owner - in this case,
+the tenant.  We should remember that all trust relationships are
+explicitly uni-directional (though this does not preclude a
+corresponding trust relationship in the other direction).  In the case
+of Enarx, it is trust relationships from the workload entity to other
+entities (primarily the host owner in this document), and maintenance
+of the tenant's trust domain, which are always the priority - any
```

Comment:
I think we should put a full-stop after the word "priority" and let "Any" be the start of a new sentence.

Reply:
Agreed.  Fixed.